ui/src/components/common/Header.js

```diff
@@ -26,15 +26,15 @@ export class Header extends React.Component {
   };
 
   onUserDropdownToggle = () => {
-    this.props.toggleHeaderDropdown();
+    this.props.toggleHeaderDropdown(); // TODO - add test coverage
```

Comment:
Should maybe use JSDoc's `@todo` style:

```js
/**
 * @todo Write the documentation.
 * @todo Implement this function.
 */
function foo() {
    // write me
}
```

http://usejsdoc.org/tags-todo.html

Reply:
@craicoverflow good point, if merging in any todo's I will look at adopting this convention. At the moment, just using TODO's to keep track of where I need to add test coverage before merging